memory/player_party_manager.py

```diff
@@ -15,6 +15,10 @@ class PlayerMovementState(Enum):
 
 
 class PlayerPartyManager:
+    NULL_POINTER = 0xFFFFFFFF
+    ITEM_OBJECT_OFFSET = 0x8
```

Comment:
`ITEM_OBJECT_OFFSET` and `ITEM_INDEX_0_ADDRESS` are potentially badly named here. They do point to a list with items, yes, but it should be possible to see what kind of items (player characters in this case?) from the names, especially if we'll need to parse other lists in this class at some point.